velox/tpch/gen/TpchGen.cpp

```diff
@@ -321,6 +321,10 @@ RowTypePtr getTableSchema(Table table) {
   return nullptr; // make gcc happy.
 }
 
+TypePtr resolveTpchColumn(Table table, const std::string& columnName) {
```

Comment:
Is there a reason for not making "table" parameter const ?

Reply:
Table is a enum (a single byte), so const ref is more expensive than passing by value.

Reply:
Sounds reasonable. Thanks.